### PR TITLE
changed to logging instead of print

### PR DESCRIPTION
**Pull request by sahilavaran**

### CODE REVIEW

File: automation/script/module.py [hunk 1]
```diff
@@ -245,7 +250,8 @@ def _run(self, i):
         # If first script run, check if can write to current directory
         if not recursion and not i.get('skip_write_test', False):
             if not can_write_to_current_directory():
-                return {'return':1, 'error':'Current directory "{}" is not writable - please change it'.format(os.getcwd())}
+                logger.error('Current directory "{}" is not writable - please change it'.format(os.getcwd()))
```
Comment (arjunsuresh) [inline automation/script/module.py:253]: We don't need to log here as the error is returned to the caller here.

Comment (sahilavaran) [reply automation/script/module.py:253]: @arjunsuresh sure i will remove that

File: automation/script/module.py [hunk 2]
```diff
@@ -562,13 +570,13 @@ def _run(self, i):
 #            print ('')
 
         if not run_state.get('tmp_silent', False):
-            print ('')
-            print (recursion_spaces + '* ' + cm_script_info)
```
[the remark this answers is not in the document]
Comment (sahilavaran) [reply automation/script/module.py:566]: @arjunsuresh yes , will do it in the logging

File: automation/script/module.py [hunk 3]
```diff
@@ -2019,7 +2007,7 @@ def _update_state_from_variations(self, i, meta, variation_tags, variations, env
                 variation_tags_string += x
 
             if verbose:
```
[the remark this answers is not in the document]
Comment (sahilavaran) [reply automation/script/module.py:2009]: @arjunsuresh  When verbose is enabled, we set the logger to DEBUG level to provide more detailed output. If verbose is False, the logger is set to INFO level, showing less detailed information.

[the remark this answers is not in the document]
Comment (arjunsuresh) [reply automation/script/module.py:2009]: Thank you. I probably missed that part. Can you please point to the place where logger is set to DEBUG level when verbose is enabled?

And any prints inside "if verbose" must be using logger.debug right?

**Comment by arjunsuresh** [comment]
closing this as we had an alternate PR.

**Pull request closed without merging**